### ISSUE TITLE
Accessibility cmd click

### ISSUE THREAD

**Issue by Phuket2** [microsoft/vscode#69719]
To be able to right click a link  (instead of cmd-clicking it on Mac) and select an option from the popup menu to **open the link**. (i.e links in terminal/editor)
This could be a setting as to whether or not this would show up in the menu as I image most people wouldn't need it.

_*I used Accessibility in the title as I only have the  **use one arm**._

**Comment by rebornix** [comment]
@Phuket2 thanks for your feedback and the suggestion of having an action in context menu when you right click sounds pretty good.

**Comment by Tyriar** [comment]
@Phuket2 do you use a standard mouse? I would think a mouse with many buttons would be ideal for this sort of disability so you could bind modifiers to individual buttons? Adding all possible modifiers to the context menu doesn't seem like a great solution as it would only work within VS Code (an OS-level feature would make more sense) and fill the context menu with many actions; ctrl+click, alt+click, shift+click.

**Comment by Tyriar** [comment]
I'm going to close this by design because:

- Links are keyboard accessible via the "Open link" command
- Adding options to the context menu would be a mess and this would only solve the problem for VS Code, not other applications

For the terminal specifically I created a new issue to capture the problem https://github.com/Microsoft/vscode/issues/69795

**Comment by rebornix** [comment]
IMO we can add the link to the hover and then users can single click to open the link.

**Comment by Tyriar** [comment]
👍 we would need to allow mousing over the hover then though, I think right now it's a setting?

**Comment by rebornix** [comment]
![mouse-over-hover](https://user-images.githubusercontent.com/876920/53905352-33368e80-3ffd-11e9-8efa-a3a949f14448.gif)

@Tyriar it's controlled by `editor.hover.sticky` and by default we can mouse over the hover as shown above.

**Comment by rebornix** [comment]
From tomorrow's Insiders you should be able to click directly in the hover to follow links

![hover-click](https://user-images.githubusercontent.com/876920/61655318-32805b00-ac73-11e9-8428-f2f3abe14bc0.gif)
